app/presenters/users_presenter.rb

```diff
@@ -48,17 +48,18 @@ def individual_article_views
     individual_courses.each do |c|
       individual_articles = c.articles
       individual_articles.each do |a|
-        article_views += a.revisions.where(user_id: @user.id).order('date ASC').first.views
+        earliest_revision = a.revisions.where(user_id: @user.id).order('date ASC').first
+        article_views += earliest_revision.views if earliest_revision.present?
       end
     end
     article_views
   end
 
   def individual_articles_created
```

Comment:
You can use a similar strategy here as in the article count to avoid overcounting new articles that occur during multiple overlapping courses: in the .each loop, collect the new article revisions, then deduplicate and count them at the end.